### PR TITLE
edit worksheet name and title

### PR DESCRIPTION
**Pull request by pbhatnagar3**
#1322 allow editing of the worksheet title and worksheet name

@percyliang @kashizui please review.

### CODE REVIEW

File: codalab/apps/web/static/js/worksheet/worksheet_interface.js [hunk 1]
```diff
@@ -311,6 +311,31 @@ var Worksheet = React.createClass({
...
+        $(document).ready(function() {
+          $('.editable-field').editable({
+            send: 'always',
+            type: 'text',
+            params: function(params) {
+              var data = {};
+              data['worksheet_uuid'] = ws_obj.state.uuid;
+              data['name'] = params['name'];
+              data['value'] = params['value'];
+              data['command'] = 'wedit -' + params['name'][0] + " '" + params['value'] + "'";
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/worksheet_interface.js]: This looks pretty dangerous to me.  Can we put this in a method in the worksheet utils class that handles making these commands?

File: codalab/apps/web/static/js/worksheet/worksheet_interface.js [hunk 2]
```diff
@@ -311,6 +311,31 @@ var Worksheet = React.createClass({
...
+                if(response.error){
+                    return response.error;
+                }
+                ws_obj.state[response.input_data['name']] = response.input_data['value'];
+                this.refs.side_panel.forceUpdate();
+
+            }.bind(this)
+          });
+        }.bind(this));
+        var name = ws_obj.state.name || 'Enter new name';
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/worksheet_interface.js]: this should not be empty

File: codalab/apps/web/static/js/worksheet/worksheet_interface.js [hunk 3]
```diff
@@ -311,6 +311,31 @@ var Worksheet = React.createClass({
...
+        var title = ws_obj.state.title || 'Enter new title';
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/worksheet_interface.js]: make this '(no title)'

File: codalab/apps/web/static/js/worksheet/worksheet_interface.js (hunk 1 above)
Comment (pbhatnagar3) [inline codalab/apps/web/static/js/worksheet/worksheet_interface.js]: sure. I will move that logic.

**Pull request merged**